src/services/redux/appointments/actions.js

```diff
@@ -0,0 +1,12 @@
+import { appointmentsSlice } from 'services/redux/appointments/reducers';
+import { RECEIVE_APPOINTMENTS, ADD_APPOINTMENTS } from 'services/redux/appointments/constants';
+
+export const receiveAppointments = (payload) => ({ type: RECEIVE_APPOINTMENTS, payload });
+export const addAppointment = (payload) => ({ type: ADD_APPOINTMENTS, payload });
+export const {
+  appointmentsReceived,
+  rejectedAppointmentsAction,
+  pendingAppointmentsAction,
+  appointmentAdded,
+  appointmentDeleted,
```

Comment:
и вот уже тут мы можем добавить окончание "Action", когда достаем экшен из слайса
```suggestion
  deleteAppointment as deleteAppointmentAction 
```